<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Kč</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Euro</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dolar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>loket</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sah</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pid</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>palec</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>stopa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>libra</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>unce</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kamen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gram</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kilogram</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>joule</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kalorie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>elektronvolt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kilowatthodina</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Kč</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dolar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Euro</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>loket</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sah</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pid</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>palec</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>stopa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>libra</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>unce</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kamen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gram</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kilogram</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>joule</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kalorie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>elektronvolt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>kilowatthodina</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>převod</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>na</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>104</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Měna</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
